packages/react-components/src/molecules/__tests__/EventDescription.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import { createEventResponse } from '@asap-hub/fixtures';
+
+import EventDescription from '../EventDescription';
+
+jest.mock('../../localization');
```

Comment:
y dis

Reply:
I wondered this too and then didn't remove it xD